app/controllers/hyrax/depositors_controller.rb

```diff
@@ -15,14 +14,18 @@ def index
     end
 
     def create
-      grantor = authorize_and_return_grantor
-      grantee = ::User.from_url_component(params[:grantee_id])
-      if grantor.can_receive_deposits_from.include?(grantee)
-        head :ok
+      if (params[:user_id].sub '.', '-dot-') == (params[:grantee_id].sub '.', '-dot-')
```

Comment:
Can you explain why the string substitution is necessary here?

Reply:
@cjcolvar params[:user_id] comes in with a "-dot-", so just to safe, I do the substitution to both sides.

(byebug) params[:user_id]
"blancoj@umich-dot-edu"
(byebug) params[:grantee_id]
"user1@umich.edu"

Reply:
Ok.  How about going the opposite direction using `::User.from_url_component(params[:user_id])` which is used on line 41?
```suggestion
      if (params[:grantee_id] == ::User.from_url_component(params[:user_id]))
```